engines/director/cast.cpp

```diff
@@ -110,7 +110,7 @@ Cast::~Cast() {
 
 CastMember *Cast::getCastMember(int castId, bool load) {
 	CastMember *result = nullptr;
-
+	CastMember *subfilmloop = nullptr;
```

Comment:
This declaration seems to me very early, You are not using this variable beyond the `if()`